modules/oe_content_consultation/config/install/field.field.node.oe_consultation.oe_summary.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_summary
+    - node.type.oe_consultation
+  module:
+    - text
+id: node.oe_consultation.oe_summary
+field_name: oe_summary
+entity_type: node
+bundle: oe_consultation
+label: Introduction
+description: ''
```

Comment:
We are missing the description. See the use of this AND all the other common fields and make them consistent with the rest of the CTs.